src/main/kotlin/com/cognifide/gradle/aem/common/instance/local/QuickstartResolver.kt

```diff
@@ -35,7 +35,7 @@ class QuickstartResolver(private val manager: LocalInstanceManager) {
      * URI pointing to AEM self-extractable JAR containing 'crx-quickstart'.
      */
     val jarUrl = aem.obj.string {
-        convention(distUrl.map { it.takeIf { it.endsWith(".jar") } })
+        convention(distUrl.map { it.takeIf { it.endsWith(".jar") }.toString() })
```

Comment:
your fix is wrong; kotlin type system is too agressive here and we are unable to avoid that one particular error.

however maybe you could find the way to annotate it to skip displaying that compile time error?

Reply:
I added `@Suppress("NULLABILITY_MISMATCH_BASED_ON_JAVA_ANNOTATIONS")`

Reply:
cool